src/main/java/org/asamk/signal/manager/Manager.java

```diff
@@ -204,6 +205,14 @@ public void updateAccountAttributes() throws IOException {
         accountManager.setAccountAttributes(account.getSignalingKey(), account.getSignalProtocolStore().getLocalRegistrationId(), true, account.getRegistrationLockPin(), getSelfUnidentifiedAccessKey(), false);
     }
 
+    public void setProfileName(String name) throws IOException {
+        accountManager.setProfileName(account.getProfileKey(), name);
+    }
+
+    public void setProfileAvatar(File avatar) throws IOException {
+        accountManager.setProfileAvatar(account.getProfileKey(), Utils.createStreamDetailsFromFile(avatar));
+    }
+
```

Comment:
Please add a third method here `removeProfileAvatar`, that pass `null` as streamDetails to remove the avatar.